### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ciaranschutte**
- updates to Docusaurus version 2.0.0-alpha.58
  - brings typescript support (will be added in separate ticket)
  - removes need for svg plugin
- fixes styles/components that broke during upgrade
- couple of components "swizzled" ie copied from Docusaurus package to be customised

### CODE REVIEW

File: website/src/theme/CodeBlock/index.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```
Comment (hlminh2000) [inline website/src/theme/CodeBlock/index.js:6]: for these copied code, do we wanna put our copy right instead? Then add a blob mentioning where it was copied from both for documentation and copyright attribution back to Facebook 👍

Comment (ciaranschutte) [reply website/src/theme/CodeBlock/index.js:6]: @hlminh2000 for sure! Do you know if there is a specific format I need to use for attribution?

Comment (hlminh2000) [reply website/src/theme/CodeBlock/index.js:6]: maybe something like `This code was originally copied from Facebook at ___ and modified under MIT license`?

**Comment by ciaranschutte** [comment]
@hlminh2000 updated copyright on relevant components. Other diffs are prettier

**Pull request merged**